src/main/webapp/data-visualization.js

```diff
@@ -296,3 +302,47 @@ function setStyle(isCountyQuery) {
     amChartsDiv.style.display = 'block';
   }
 }
+
+function initDataTable(isCountyQuery) {
+  const table = document.createElement('table');
+  table.innerHTML = '';
+  const tableRow = document.createElement('tr');
+  const region = document.createElement('th');
+  const value = document.createElement('th');
+  region.innerText = isCountyQuery ? 'County' : 'State';
+  value.innerText = 'Population';
+  tableRow.appendChild(region);
+  tableRow.appendChild(value);
+  table.appendChild(tableRow);
+  return table;
+}
+
+/**
+ * Create and return an HTML table using the data in dataArray.
+ * Uses first row in dataArray as table headers.
+ */
+function appendRowToDataTable(table, locationName, population) {
+  const tableRow = document.createElement('tr');
+  const region = document.createElement('td');
+  const value = document.createElement('td');
+  region.innerText = locationName;
+  value.innerText = population;
+  tableRow.appendChild(region);
+  tableRow.appendChild(value);
+  table.appendChild(tableRow);
```

Comment:
Some of this looks very similar to initDataTable; is there a way to write those lines of code just once?